client/app/lib/components/ConfirmationDialog.jsx

```diff
@@ -16,11 +16,11 @@ class ConfirmationDialog extends React.Component {
       open,
       onCancel,
       onConfirm,
-      onConfirmCustom,
+      onConfirmSecondary,
       message,
       cancelButtonText,
       confirmButtonText,
-      confirmButtonCustomText,
+      confirmButtonSecondaryText,
```

Comment:
Nit: Should be `confirmSecondaryButtonText`, since it's not the secondary text of the confirm button, but the button text of the secondary confirm.